.github/workflows/ci.yml

```diff
@@ -271,6 +271,23 @@ jobs:
           target: aarch64-apple-darwin
           args: --release -i python3.9 --no-sdist -m examples/maturin-starter/Cargo.toml
 
+      - name: Test cross compile to Windows
+        if: ${{ matrix.platform.os == 'ubuntu-latest' && matrix.python-version == '3.8' }}
+        env:
+          XWIN_ARCH: x86_64
+        run: |
+          sudo apt-get install -y mingw-w64 llvm
+          rustup target add x86_64-pc-windows-gnu x86_64-pc-windows-msvc
+          which cargo-xwin > /dev/null || cargo install cargo-xwin
+          cargo build --manifest-path examples/maturin-starter/Cargo.toml --features abi3 --target x86_64-pc-windows-gnu
+          cargo xwin build --manifest-path examples/maturin-starter/Cargo.toml --features abi3 --target x86_64-pc-windows-msvc
```

Comment:
Plain `cargo build` does not work because of missing Windows CRT and SDK files for example `kernel32.lib`.

https://github.com/PyO3/pyo3/runs/5962060548?check_suite_focus=true

Reply:
Thanks @messense! How come this doesn't need to refer to the new feature?

Reply:
The new feature is enabled by `maturin-starter`'s `abi3` feature, see https://github.com/PyO3/pyo3/pull/2282/files#diff-074902230e9e60917cbf6dcd41751d36ff5a6f3afb24bbd5a62317eeebc8a2b6R14

Reply:
Aha missed that, thanks!